src/pages/[guild]/index.tsx

```diff
@@ -122,126 +122,173 @@ const GuildPage = (): JSX.Element => {
 
       {featureFlags?.includes("ONGOING_ISSUES") && <DynamicOngoingIssuesBanner />}
 
-      <Layout
-        title={<GuildName {...{ name, tags }} />}
-        ogTitle={name}
-        textColor={textColor}
-        ogDescription={description}
-        description={
-          (description || Object.keys(socialLinks ?? {}).length > 0) && (
-            <>
+      <Layout>
+        <LayoutHero>
+          <LayoutBanner>
+            {localBackgroundImage ? (
+              <Image
+                src={localBackgroundImage}
+                alt="Guild background image"
+                priority
+                fill
+                sizes="100vw"
+                style={{
+                  filter: "brightness(30%)",
+                  objectFit: "cover",
+                }}
+              />
+            ) : (
+              <div
+                className={cn("h-full w-full", {
+                  // TODO: rework ThemeContext & don't use Chakra related colors
+                  "opacity-50": textColor !== "primary.800",
+                })}
+                style={{
+                  backgroundColor: localThemeColor,
+                }}
+              ></div>
+            )}
+          </LayoutBanner>
+
+          <Header />
+
+          <LayoutContainer className="-mb-14 mt-6">
+            <BackButton />
+          </LayoutContainer>
+
+          <LayoutHeadline className="pt-8">
+            {imageUrl && (
+              <Avatar
+                className={cn("row-span-2 size-20 md:size-24", {
+                  // TODO rework ThemeContext & use a non-Chakra CSS variable
+                  "bg-[var(--chakra-colors-primary-800)]":
+                    textColor === "primary.800",
+                })}
+              >
+                <AvatarImage
+                  src={imageUrl}
+                  alt={`${name} logo`}
+                  width={96}
+                  height={96}
+                />
+                <AvatarFallback>
+                  <Skeleton className="size-full" />
+                </AvatarFallback>
+              </Avatar>
+            )}
+            <div className="flex flex-col gap-1">
+              <div className="flex items-start gap-1 sm:gap-1.5">
+                <LayoutTitle
+                  className={cn(
+                    "line-clamp-1 break-all leading-tight sm:leading-tight",
+                    {
+                      // TODO rework ThemeContext & use a non-Chakra CSS variable
+                      "text-[var(--chakra-colors-primary-800)]":
+                        textColor === "primary.800",
+                    }
+                  )}
+                >
+                  {name}
+                </LayoutTitle>
+                <CheckMark className="mt-2.5 size-5 shrink-0 sm:mt-4 sm:size-6" />
+              </div>
+
+              <Badge
```

Comment:
I think we should use the common MemberCount component here. I know that's not Tailwind yet, we'll be able to migrate it there once and it'll change everywhere